### PR TITLE
Fix: Issues upserting by ID and specifying nillable fields

### PR DESCRIPTION
**Pull request by ikrasnykh**
Since every object extends sObject, need to append sObject elements to all native and custom objects

### CODE REVIEW

File: src/Phpforce/SoapClient/Soap/SoapClient.php [hunk 1]
```diff
@@ -40,6 +40,12 @@ public function getSoapTypes()
                     preg_match('/\s* (.*) (.*);/', $line, $matches);
                     $properties[$matches[2]] = $matches[1];
                 }
+
+                // IK 4.4.2015: Since every object extends sObject, need to append sObject elements to all native and custom objects
```
Comment (ddeboer) [inline src/Phpforce/SoapClient/Soap/SoapClient.php]: What does `IK` mean?

**Comment by ddeboer** [comment]
Thanks, could you please squash your commits?

File: src/Phpforce/SoapClient/Soap/SoapClient.php (hunk 1 above)
Comment (technweb) [inline src/Phpforce/SoapClient/Soap/SoapClient.php]: I'll remove that comment prefix

**Comment by technweb** [comment]
Squashed commits. Let me know if you need any additional help

**Comment by ddeboer** [comment]
Great, thanks!

**Pull request merged**